packages/wallet-connectors/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.5.0]
```

Comment:
```suggestion
## [Unreleased]

## [0.5.0]
```

But really I generally prefer setting version numbers (which in this file should include release date) in a separate "release" PR.

Reply:
The suggestion was to include `## [Unreleased]`, not replace `## [0.5.0]`. But this is also fine :)